docs/studios/unity/tutorials/first_tutorial.md

```diff
@@ -21,12 +21,32 @@ power ups (represented by blue spheres). We will cover:
 
 ![A Unity scene with a cube and a sphere](first_bot_tutorial_images/tutorial_0_start.png)
 
+## Create an account on Regression Games
+
+The first step is to create an account on Regression Games. Visit the Regression Games site (usually
+this is [https://play.regression.gg](https://play.regression.gg), but it may be different for your organization),
+and click "sign up". Enter a username, email, and password. Once you log in, you will be prompted to connect
+a GitHub account. After connecting your account, the Regression Games platform will be able to pull and live reload
+bots through GitHub repositories.
+
+You will also be prompted to enter a Minecraft username - you can put anything for this field (it will be removed
+soon as part of our new studio-centric experience).
+
 ## Download the sample scene
 
 We have a starting scene that you can play around with to get started. Download that project 
-[here](https://github.com/Regression-Games/RGUnitySample). This scene features a prefab for a character and a power 
-up type, as well as three scripts; one for controlling the game loop, one for controlling the character, and one for 
-controlling the power up logic.
+[here](https://github.com/Regression-Games/RGUnitySample/tree/starter) or clone it using GitHub. **Make sure to
+download or clone the `starter` branch**, which serves as the starting point for this tutorial. For
+the completed tutorial, see the `main` branch.
```

Comment:
I was thinking about that, and actually had that implemented first, but then I figured it would be a better experience this way in case someone is just looking at the repo for information / just to take a look quickly. I thought it would be better to be able to see the integration code right away.

Reply:
Okay, I see your point